app/views/layouts/application.html.erb

```diff
@@ -11,6 +11,9 @@
   </head>
 
   <body>
+    <% flash.each do |name, msg| %>
+      <%= msg %>
+    <% end %>
```

Comment:
let's put this into a "_flashes" partial, and render it here.

just like you render a `form` partial